src/Uno.UI/UI/Xaml/Controls/CalendarView/CalendarViewGeneratorHost.cs

```diff
@@ -350,7 +350,8 @@ internal DateTime GetDateAt(uint index)
 		internal int CalculateOffsetFromMinDate(DateTime date)
 		{
 			var pIndex = 0;
-			DateTime estimatedDate = m_lastVisitedDateAndIndex.first; //.ToUniversalTime(); - UNO
+			DateTime estimatedDate = m_lastVisitedDateAndIndex.first.ToUniversalTime();
```

Comment:
@Youssef1313  Why did we add this in the first place? Was it for this particular crash?

Reply:
Yes. PR: https://github.com/unoplatform/uno/pull/12179

Reply:
On which platform does this fix the issue? I'm not sure to follow how to reproduce this. Is there a sample that demonstrates it?

Reply:
@jeromelaban I was able to write a test.